### PR TITLE
Feature/text updates

### PR DESCRIPTION
**Pull request by jbranigan**
Updated text based on IMLS comments.

### CODE REVIEW

File: app/scripts/views/about/about-partial.html [hunk 1]
```diff
@@ -22,28 +22,29 @@
...
+                    <p>MuseumStat provides community leaders with important indicators of community health and well-being, easily accessible through maps, charts, and graphs. A wide range of museums are identified as community assets and potential partners for programs and services.</p>
 
                     <div class="content-title">
                         <h3>For Researchers</h3>
                     </div>
-                    <p>MuseumStat provides researchers, advocates, and policymakers with important, objective data to understand the scope, reach, and impact of museums in communities. Data is visualized on-screen and provided for download to allow for the widest possible use and analyses.</p>
+                    <p>MuseumStat provides researchers, advocates, and policymakers with important data to understand the potential  reach of museums in communities. Data is visualized on-screen and provided for download to allow for the widest possible use and analyses.</p>
```
Comment (CloudNiner) [inline app/scripts/views/about/about-partial.html]: Extra space in `potential  reach`

**Comment by CloudNiner** [comment]
LGTM

After fixing the spacing in that one spot, I'll merge and update the site.

**Comment by nevillevak** [comment]
Thanks!

________________________________
Neville K. Vakharia
Assistant Professor and Research Director
Drexel University, Westphal College of Media Arts & Design
Department of Arts & Entertainment Enterprise
215.571.4439, nvakharia@drexel.edu

Office Hours:
Tuesdays and Thursdays, 2:00 – 5:00
By appointment only

**Comment by jbranigan** [comment]
Should be all set, @CloudNiner

**Pull request merged**